src/NzbDrone.Api/Qualities/QualityDefinitionResource.cs

```diff
@@ -53,7 +53,7 @@ public static QualityDefinition ToModel(this QualityDefinitionResource resource)
                 Weight = resource.Weight,
 
                 MinSize = resource.MinSize,
-                MaxSize = resource.MaxSize
+                MaxSize = resource.MaxSize,
```

Comment:
Needed?

Reply:
Probably not, also a leftover from before, but IMO an extra comma is always good, so you don't forget it if you add something.